### PR TITLE
Vendor update for accounts

### PR DESCRIPTION
**Pull request by bquenin**
Adding github.com/hlandau/passlib
Adding github.com/dgrijalva/jwt-go
Adding github.com/holys/safe

### CODE REVIEW

**Review by freignat91 — APPROVED** [review]
LGTM

File: glide.lock [hunk 1]
```diff
@@ -250,9 +256,11 @@ imports:
   - internal/bitbucket.org/ww/goautoneg
   - model
 - name: github.com/prometheus/procfs
-  version: 1878d9fbb537119d24b21ca07effd591627cd160
+  version: 3c6d1cd72106fb2c83336c7766b365da0cf9574e
+  subpackages:
+  - xfs
 - name: github.com/Sirupsen/logrus
-  version: 55eb11d21d2a31a3cc93838241d04800f52e823d
+  version: 3f603f494d61c73457fb234161d8982b9f0f0b71
```
Comment (ndegory) [inline glide.lock]: when I run the cleanall and install-deps rules, this change is reverted, did you use the same rules to update the glide.lock? (in particular the cleanall rule)

File: vendor/github.com/hlandau/passlib/README.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
+passlib for go
```
Comment (ndegory) [inline vendor/github.com/hlandau/passlib/README.md:1]: That file (and a bunch of other files from gopkg.in/hlandau package) are deleted when I run the install-deps.
We should have the same result, that's troublesome!

**Review by ndegory — CHANGES_REQUESTED** [review]
I'm worried by the differences I get when I run the install-deps rule with the same glide.yaml file, regardless of recent commits in dependencies that are normal differences.

File: vendor/github.com/hlandau/passlib/README.md (hunk 2 above)
Comment (bquenin) [reply vendor/github.com/hlandau/passlib/README.md:1]: I'm not sure I'm following, I just ran:
`glide get github.com/hlandau/passlib`
`glide get github.com/dgrijalva/jwt-go`
`glide get github.com/holys/safe`


Is there anything else I should pay attention to ?

Comment (bquenin) [reply vendor/github.com/hlandau/passlib/README.md:1]: These discrepancies are due to the fact I didn't update the vendors globally but only added the necessary ones (i.e. the one mentioned in the PR comment)
Running:
```
make -f Makefile.refactor.make cleanall
make -f Makefile.refactor.make install-deps
```
updates the vendor folder globally, which is not my intent.

Comment (bquenin) [reply vendor/github.com/hlandau/passlib/README.md:1]: Forced push with:
```
make -f Makefile.refactor.make cleanall
make -f Makefile.refactor.make install-deps
```

**Review by ndegory — APPROVED** [review]

**Pull request merged**